activation/handler_v1.go

```diff
@@ -577,6 +577,7 @@ func (h *HandlerV1) storeAtx(
 	ctx context.Context,
 	atx *types.ActivationTx,
 	watx *wire.ActivationTxV1,
+	blob []byte,
```

Comment:
`blob` doesn't need to be passed all the way from `processATX` to `storeAtx`. I think it would be enough to only pass `*wire.ActivationTxV1` and call `Blob()` on it when storing.

Reply:
That would require re-encoding of the `watx`, which can be avoided by passing the blob all along.

Reply:
not necessarily: we could move `decodeATX` into the `wire` package and set the bytes as private field on the `wire.AtxV1/V2` type and `Blob()` just return that instead of re-encoding. Wdyt?

Reply:
I'd prefer to pass it explicitly rather than to have a blob field on the wire type that might or might not be initialized.

Reply:
We already have a `Blob()` method on the type we call that and it returns the contents of the private field or lazily initializes that private field first if its `nil`?

Reply:
Similar to how we do it with the ID of the ATX

Reply:
Updated as proposed.